scripts/runcode.jl

```diff
@@ -311,16 +311,16 @@ julia> chamber(composition, end_time, log_volume_km3, InitialConc_H2O, InitialCo
 julia> composition = Mafic()
 julia> end_time = 3e9
 julia> log_volume_km3 = 0.2
-julia> InitialConc_H2O = 0.01
+julia> InitialConc_H2O = [0.01, 0.012]
 julia> InitialConc_CO2 = 0.001
 julia> log_vfr = -3.3
-julia> depth = 8e3
+julia> depth = [7e3, 8e3]
 julia> output_dirname = "MyDirname"
 
 julia> chamber(composition, end_time, log_volume_km3, InitialConc_H2O, InitialConc_CO2, log_vfr, depth, output_dirname)
 ```
 """
-function chamber(
+function arraychamber(
```

Comment:
Just use the same name. This feature of the language is called [multiple dispatch](https://en.wikipedia.org/wiki/Multiple_dispatch).
```suggestion
function chamber(
```